provider-ui/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "dev": "nodemon ui_dev_server.js",
-    "lint": "eslint --fix .",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix",
```

Comment:
Specify the path in this command, i.e, `eslint --fix .`
Also, keep the command name as it is for the `--fix` option, rather update the new command as `checklint: eslint .`

Reply:
And, update the same in the Makefile.

Reply:
Those cache needs to be retrived intead of fresh install, additionally, if it is not required to cache then better not cache it.

Reply:
Yeah, but even the current workflow (without this change) can be optimized to remove recurring caching of the node_modules. WDYT?

Reply:
Yeah it can be.